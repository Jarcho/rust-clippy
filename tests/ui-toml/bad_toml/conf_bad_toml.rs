//@error-in-other-file: error reading `clippy.toml`: expected `.`, `=`

fn main() {}
