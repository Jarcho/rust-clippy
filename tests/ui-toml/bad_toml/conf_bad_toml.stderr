error: error reading `clippy.toml`: expected `.`, `=`
  --> $DIR/tests/ui-toml/bad_toml/clippy.toml:1:4
   |
LL | fn this_is_obviously(not: a, toml: file) {
   |    ^

error: aborting due to 1 previous error

